cli/loader/Cargo.toml

```diff
@@ -14,6 +14,7 @@ repository = "https://github.com/tree-sitter/tree-sitter"
 anyhow = "1.0"
 cc = "^1.0.58"
 dirs = "3.0"
+home = "0.5.3"
```

Comment:
Small nit: We've already taken a dependency on `dirs`, which provides a [`home_dir`](https://docs.rs/dirs/3.0.2/dirs/fn.home_dir.html) function that we can use. Can we use that instead of taking a dependency on the `home` crate?

Reply:
Sure, let me fix and force push.

Reply:
Pushed :)